railcard.css

```diff
@@ -9,22 +9,24 @@
     flex-direction: row;
     align-items: center;
     justify-content: center;
-    margin: 10px;
-    width: 80%;
+    margin-top: 2%;
+    margin-bottom: 100px;
+    width: 100%;
+    padding: 20px 0px;
     flex-wrap: wrap;
 }
 #railcard #railcard_headbar .imgdiv {
     display: flex;
     align-items: center;
     justify-content: center;
-    width: 64px;
-    height: 64px;
+    width: 150px;
+    height: 150px;
 }
 #railcard #railcard_headbar .imgdiv:hover {
     cursor: pointer;
 }
 #railcard #railcard_headbar img {
-    width: 48px;
+    width: 120px;
```

Comment:
120px is a bit too large for small screens, may be you have a large screen😂

<img width="923" alt="display on macbook" src="https://github.com/LogCreative/StarRailCard-web/assets/61653082/875d20cc-241c-43d3-8981-3fbb6e1b9f0d">